app/components/calendar-input.js

```diff
@@ -7,6 +7,7 @@ const MIN_DATE = moment()
 const DEFAULT_PICKADATE_CONFIG = {
   selectMonths: true,
   selectYears: true,
+  format: "dd/mmm/yyyy",
```

Comment:
I notice there's `format` and `formatSubmit` 

I feel there should only be one, which is set by whoever uses the component 🤔